src/traces/scattergl/index.js

```diff
@@ -486,6 +486,22 @@ function sceneUpdate(gd, subplot) {
     if(!subplot._scene) {
         scene = subplot._scene = Lib.extendFlat({}, reset, first);
 
+        // apply new option to all regl components (used on drag)
+        scene.update = function update(opt) {
```

Comment:
cc @dfcreative you were right! `scene.update` is still used on drag. Added spy+assertion below to  :lock_with_ink_pen: this thing down.